src/components/Heading/Heading.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+import { DefaultProps } from '../../types';
+
+import './Heading.scss';
+
+export interface HeadingProps extends DefaultProps {
+	type: 'h1' | 'h2' | 'h3' | 'h4';
+	children: 'string' | ReactNode;
+}
+
+export const Heading: FunctionComponent<HeadingProps> = ({ className, children, ...props }) => (
+	<props.type className={classNames(className, `c-${props.type}`)}>{children}</props.type>
```

Comment:
Waarom doe je `props.type`?

Reply:
Anders lukt het niet om een variable als tag te gebruiken for some reason. Typescript heeft het niet graag omdat het geen inner children verwacht dan, ook al specify ik ze. 🤷‍♂

Reply:
Moet een hoofdletter zijn dus je kan dit in je destructure doen `type: Type`